docs/content.zh/docs/dev/table/sql-gateway/rest.md

```diff
@@ -94,7 +94,12 @@ Endpoint Options
 REST API
 ----------------
 
-[OpenAPI specification]({{< ref_static "generated/rest_v1_sql_gateway.yml" >}})
+The available OpenAPI specification as follows. The default version is v2.
```

Comment:
Do you have any plans to translate the doc?

Reply:
I think we can leave the work for the future 0.0